### DISCUSSION TITLE
How to disable opening URL to vcpkg

### DISCUSSION THREAD

**Discussion by aenachescu** [microsoft/vscode-cpptools#7246] [Q&A]
Hello,

When an included file is not found after I save the file the extension opens an URL to vcpkg. This is the URL that is open: https://docs.microsoft.com/en-us/cpp/build/vcpkg?view=msvc-160
Here is an example:
![image](https://user-images.githubusercontent.com/6330622/112748116-74be7b80-8fc2-11eb-80bd-98c7226221e4.png)

If i disable this extension everything is ok. If i disable all extensions, except this extension, the URL still opens on file save.
Can i disable this behaviour?

VsCode details
Version: 1.54.3
Commit: 2b9aebd5354a3629c3aba0a5f5df49f43d6689f8
Date: 2021-03-15T10:55:24.277Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 5.4.0-70-generic

Extension version: v1.3.0-insiders2

**Comment by sean-mcmanus** [comment]
That should only occur when you click the code action shown in this this screenshot. Do you know how you're getting that to occur in another scenario?

![image](https://user-images.githubusercontent.com/19859882/112872942-d390fd00-9075-11eb-87d0-6bd5e0edbf56.png)

**Reply by aenachescu** [reply]
Hi Sean. Sorry for the late response. The link is open when i click on the action in the screenshot but it is also open when i press ctrl+s (save file). I took a look at the extension code and you right, the link should be open only if the action from screenshot is triggered. I can't understand what the heck is happening on my machine. I've had this problem for a long time and I ignored it but now I really want to fix it.
I use vscode on 4 laptops/pc and i have this problem on all of them (but i have only one account so the settings/extensions/etc are the same). I don't have any shortcut set for the C_Cpp.VcpkgOnlineHelpSuggested command. Maybe the save file command triggers some quick fixes and that is the reason why your command is executed? This problem always reproduce and it has become frustrating. Do you need logs or something else in order to investigate the problem?

**Reply by sean-mcmanus** [reply]
Can you set the C_Cpp.vcpkg.enabled setting to false as a workaround? My guess is that it's some VS Code bug, since I debugged the small amount of code involved on our side and I don't see what could be going wrong from us.

**Reply by aenachescu** [reply]
I disabled that setting and now it is ok. Thank you!
